src/devices/LiquidLevelSwitch/README.md

```diff
@@ -0,0 +1,17 @@
+# Digital liquid level switch
+
+Digital liquid level switches are devices that can detect the presence of liquid/water. GPIO can be used to communicate with the devices.
+
+## Devices
+
+The implementation supports the following devices:
+
+- LLC200D3SH sensor ([Datasheet](https://cdn-shop.adafruit.com/product-files/3397/3397_datasheet_actual.pdf))
```

Comment:
if you remove `abstract` then you can mention that any switch with single pin output will do as long as it's configured